scripts/clustering/clustering.py

```diff
@@ -55,14 +65,14 @@ def in_same_cluster(s1, s2, clusters):
         
     return False            
 
-def get_tanimoto_coeffient_by_cluster(smiles, clusters):
+def get_tanimoto_coeffient_by_cluster(smiles, clusters, fp_type):
```

Comment:
There is also a usage of compare two smiles in recluster_singletons. I get errors with this. It needs to be added in.